www/%username/widgets/index.html.spt

```diff
@@ -28,22 +28,26 @@ profile_url = participant.url()
     <a href="{{ profile_url }}donate"><img src="/assets/widgets/donate.svg"></a>
 
     <h3>{{ _("Giving & Receiving Widgets") }}</h3>
+    % if participant.hide_receiving and participant.hide_giving
+        <p>{{ _("To enable widgets, uncheck 'hide' giving and/or receiving under Privacy in <a href=\"{0}\">your settings</a>."|safe, profile_url) }}</p>
```

Comment:
This is incorrect, the HTML should be in the arguments, not in the source string:

```
_("... {link_open}your settings{link_close}.",
  link_open='<a href="%s">'|safe % url,
  link_close='</a>'|safe)
```